profile/rwahs.xml

```diff
@@ -2729,14 +3420,21 @@
               <bundle>ca_occurrences</bundle>
               <settings>
                 <setting name="label" locale="en_AU">Subject List</setting>
-                <setting name="restrict_to_type">subject</setting>
+                <setting name="readonly"/>
```

Comment:
This is a fix for https://youtrack.gaiaresources.com.au/issue/RWAHS-144 - same change was necessary for Library further down, so included that fix in this PR